pages/api/projects/index.js

```diff
@@ -7,6 +7,17 @@ export default async function handler(request, response) {
   if (request.method === "GET") {
     const projects = await Project.find();
     return response.status(200).json(projects);
+  }
+  if (request.method === "POST") {
+    try {
+      const projectData = request.body;
+      console.log("in api: ", projectData);
```

Comment:
```suggestion
```
Please no console.logs inside of code you're trying to get ready to be pushed to production.